pyproject.toml

```diff
@@ -0,0 +1,68 @@
+[build-system]
+requires = ["setuptools", "wheel"]
+build-backend = "setuptools.build_meta"
+
+[project]
+name = "tomato-alicat"
+version = "0.1.0"
```

Comment:
This can be set dynamically, if you use `setuptools-git-versioning` for example. See how it's done here:

https://github.com/dgbowl/tomato-bronkhorst/blob/main/pyproject.toml